src/components/Label/Label.stories.mdx

```diff
@@ -31,14 +31,15 @@ import { getCategory } from "../../utils/componentCategories";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.0.10`   |
-| Latest            | `1.0.6`    |
+| Latest            | `1.2.0`    |
 
 ## Table of Contents
 
 - [Overview](#overview)
 - [Component Props](#component-props)
 - [Accessibility](#accessibility)
 - [isRequired helper text](#isrequired-helper-text)
+- [requiredLabelText](#requiredlabeltext-to-customize-helper-text)
```

Comment:
The text inside the brackets `[]` should be the same as the hash inside the parenthesis `()`. Let's rename the text to `requiredLabelText to customize required label`. The "helper text" is already a concept for form components and saying the same here might be confusing.

Reply:
changed